tsfile/src/main/java/org/apache/iotdb/tsfile/read/filter/basic/UnaryFilter.java

```diff
@@ -51,4 +60,36 @@ public FilterType getFilterType() {
 
   @Override
   public abstract Filter clone();
+
+  @Override
+  public void serialize(DataOutputStream outputStream) {
+    try {
+      outputStream.write(getSerializeId().ordinal());
+      outputStream.write(filterType.ordinal());
+      ReadWriteIOUtils.writeObject(value, outputStream);
+    } catch (IOException ignored) {
+      // ignored
```

Comment:
log?

Reply:
This DataOutputStream is backed-up with a ByteArrayOutputStream, technically, there is no chance that an exception will be thrown.